plugin/core/src/test/groovy/com/novoda/gradle/release/ReleasePluginTest.groovy

```diff
@@ -40,12 +40,17 @@ class ReleasePluginTest {
                 BuildConfiguration.forAndroid('4.2', '3.0.0'),
                 BuildConfiguration.forAndroid('4.3', '3.0.0'),
                 BuildConfiguration.forAndroid('4.4', '3.1.0'),
+                BuildConfiguration.forAndroid('4.5', '3.1.0'),
+                BuildConfiguration.forAndroid('4.6', '3.2.0'),
+                BuildConfiguration.forAndroid('4.7', '3.2.0'),
```

Comment:
Apparently the internal API has changed again in Gradle 4.8, so we will need to address that via separate compat project.